pyproject.toml

```diff
@@ -14,19 +14,19 @@ classifiers = [
     "Operating System :: POSIX",
     "Programming Language :: Python :: 3",
     "Programming Language :: Python :: 3 :: Only",
-    "Programming Language :: Python :: 3.10",
+    "Programming Language :: Python :: 3.11",
 ]
 dependencies = [
     "asdf >=3.3.0",
     "astropy >=5.3",
     "crds >=11.16.16",
     "defusedxml >=0.5.0",
     "galsim >=2.5.1",
-    "rad >=0.22.0",
-    "roman_datamodels >=0.22.0",
+    "rad >=0.23.0",
+    "roman_datamodels >=0.23.0",
     "gwcs >=0.19.0",
     "jsonschema >=4.8",
-    "numpy >=1.22",
+    "numpy >=1.24",
```

Comment:
not sure this is the cause but change line 5 for the required python  to be ">=3.11"